.github/workflows/e2e-tests.yml

```diff
@@ -63,6 +63,49 @@ jobs:
           yarn jest tests/format ${{ matrix.additional-options }}
         working-directory: prettier
 
+  prettier-next:
+    name: Prettier-next
+    strategy:
+      matrix:
+        additional-options: ["--maxWorkers=2", "--runInBand"]
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: jest-light-runner
+
+      - name: Checkout "prettier-next"
+        uses: actions/checkout@v4
+        with:
+          repository: prettier/prettier
+          ref: next
+          path: prettier
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
+
+      - name: Build
+        run: |
+          yarn install --immutable
+          yarn build
+        working-directory: prettier
+
+      - name: Update jest-light-runner
+        run: |
+          yarn up jest-light-runner@file:../jest-light-runner
+        working-directory: prettier
+
+      - name: Run tests
+        env:
+          NODE_ENV: production
```

Comment:
Its cause.🤦‍♂️

Reply:
... Thanks!

Reply:
No wonder `--runInBand` doesn't collect anything either, I was so stupid.

Reply:
It doesn't matter.:)